app/assets/javascripts/application.js

```diff
@@ -56,3 +56,10 @@
 //= require keybindings.js
 //= require realtime_username_validation.js
 //= require cable.js
+
+Notification.requestPermission().then(function (permission) {
```

Comment:
Hi! I guess I'd prefer this to be in a sub-file like `notifications.js` or could it fit in the `cable.js` file? Just because this is mostly a file for concatenating things. Thanks!

Reply:
Done! Let's merge and test this out.